.changeset/big-pets-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/preact': patch
+---
+
+Added preact-ssr-prepass library to preact integraiont
```

Comment:
```suggestion
Suspense on the server with Preact

Adds `preact-ssr-prepass` library to the official Preact integration.
```
If we're looking more closely at changesets that might become part of the automated blog post, we could do something like this for a header. If not, just use the corrections to the second sentence!  :smile: